Backpack/Map/Classes/BPKMapAnnotationViewCalloutView.m

```diff
@@ -99,43 +96,46 @@ - (void)setContentColor:(UIColor *)contentColor {
 }
 
 -(void)updateStyle {
-    if(!self.annotationView.enabled) {
-        self.label.fontStyle = BPKFontStyleTextSmEmphasized;
-        self.backgroundView.backgroundColor = [BPKColor dynamicColorWithLightVariant:BPKColor.white darkVariant:BPKColor.blackTint03];
-        self.contentView.backgroundColor = BPKColor.clear;
-        self.contentColor = [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint04 darkVariant:BPKColor.blackTint05];
-        self.contentView.layer.borderColor = BPKColor.clear.CGColor;
-        [self updateIcon];
-        return;
-    }
-
-    if(self.annotationView.selected) {
-        self.label.fontStyle = BPKFontStyleTextBaseEmphasized;
-        self.backgroundView.backgroundColor = BPKColor.skyBlue;
-        self.contentView.backgroundColor = BPKColor.white;
-        self.contentColor = BPKColor.skyBlue;
-        self.contentView.layer.borderColor = BPKColor.skyBlue.CGColor;
-        [self updateIcon];
-        return;
-    }
+    BPKFontStyle fontStyle = BPKFontStyleTextBase;
+    UIColor *backgroundColor = nil;
+    UIColor *contentBackgroundColor = nil;
+    UIColor *contentColor = nil;
+    UIColor *borderColor = nil;
 
-    if (self.annotationView.hasBeenSelected) {
-        self.label.fontStyle = BPKFontStyleTextSmEmphasized;
-        self.backgroundView.backgroundColor = BPKColor.skyBlueTint03;
-        self.contentView.backgroundColor = BPKColor.clear;
-        self.contentColor = BPKColor.skyBlue;
-        self.contentView.layer.borderColor = BPKColor.clear.CGColor;
-        [self updateIcon];
-        return;
+    if(!self.annotationView.enabled) {
+        fontStyle = BPKFontStyleTextSmEmphasized;
+        backgroundColor = [BPKColor dynamicColorWithLightVariant:BPKColor.white darkVariant:BPKColor.blackTint03];
+        contentBackgroundColor = BPKColor.clear;
+        contentColor = [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint04 darkVariant:BPKColor.blackTint05];
+        borderColor = BPKColor.clear;
+    } else if(self.annotationView.selected) {
+        fontStyle = BPKFontStyleTextBaseEmphasized;
+        backgroundColor = BPKColor.skyBlue;
+        contentBackgroundColor = BPKColor.white;
+        contentColor = BPKColor.skyBlue;
+        borderColor = BPKColor.skyBlue;
+    } else if (self.annotationView.hasBeenSelected) {
+        fontStyle = BPKFontStyleTextSmEmphasized;
+        backgroundColor = BPKColor.skyBlueTint03;
+        contentBackgroundColor = BPKColor.clear;
+        contentColor = BPKColor.skyBlue;
+        borderColor = BPKColor.clear;
+    } else {
+        fontStyle = BPKFontStyleTextSmEmphasized;
+        backgroundColor = BPKColor.skyBlue;
+        contentBackgroundColor = BPKColor.clear;
+        contentColor = BPKColor.white;
+        borderColor = BPKColor.clear;
```

Comment:
Refactored to store the values in variables and then apply them all once at the end of the method.

This is more readable as it avoids us having to return in each case.